src/e2e-test/java/io/cdap/plugin/odp/stepsdesign/DesginTimeODP.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.odp.stepsdesign;
+
+import io.cdap.e2e.pages.actions.CdfGcsActions;
+import io.cdap.e2e.utils.SeleniumDriver;
+import io.cdap.e2e.utils.SeleniumHelper;
+import io.cdap.plugin.odp.actions.ODPActions;
+import io.cdap.plugin.odp.locators.ODPLocators;
+import io.cdap.plugin.odp.utils.CDAPUtils;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.io.IOException;
+
+/**
+ * DesginTimeODP.
+ */
+public class DesginTimeODP {
+
+  private static final String A = "a";
+
+  @Then("Connection is established")
+  public void connectionIsEstablished() throws InterruptedException {
+    CdfGcsActions.getSchema();
+    WebDriverWait wait = new WebDriverWait(SeleniumDriver.getDriver(), 20000);
+    wait.until(ExpectedConditions.numberOfElementsToBeMoreThan(By.xpath("//*[@placeholder=\"Field name\"]"), 2));
+    Assert.assertEquals(true, CDAPUtils.schemaValidation());
+  }
+
+  @When("LoadProp {string} {string} {string} {string} {string} {string} {string} {string} {string}")
+  public void loadConnection(String client, String asHost, String msServ, String systemID, String dsName,
+                             String gcsPath, String splitRow, String pkgSize, String lgrp)
+    throws IOException, InterruptedException {
+    ODPActions.getODPProperties();
+    ODPActions.enterLoadConnectionProperties(
+      client, asHost, msServ, systemID, lgrp, dsName, gcsPath, splitRow, pkgSize);
+  }
+
+  @When("User has selected Sap client macro to configure")
+  public void userHasSelectedSapClientMacroToConfigure() {
+    ODPLocators.macros.get(0).click();
+    ODPLocators.sapClient.sendKeys(Keys.chord(Keys.CONTROL, A));
```

Comment:
concerned if this operation is OS specific because in MAC we use `CMD+A` to perform this operation?
If the use case is to clear the input text and add value can we use `SeleniumHelper.replaceElementValue()` for this? @rmstar what do you think?

Reply:
done